src/elements/OcProgress.vue

```diff
@@ -1,5 +1,15 @@
 <template>
-  <progress class="uk-progress" :value="value" :max="max"></progress>
+  <progress
+    :value="value"
+    :max="max"
+    :aria-valuemax="max"
+    :aria-valuenow="value"
+    aria-busy="true"
```

Comment:
just noticed this: this assumes that we'll always use `v-if` to avoid rendering the progress bar when not busy.
if one would use `v-show` with a falsy value, would aria-busy still be in effect ?

probably not critical, I'm fine keeping it like this now

Reply:
To my understanding `aria-busy` is a value that would be in effect when just hidden, but wouldn't do any harm communication-wise as long as the progressbar is not focussed